fv3core/stencils/xtp_u.py

```diff
@@ -131,7 +175,7 @@ def __call__(self, c: FloatField, u: FloatField, flux: FloatField):
         Compute flux of kinetic energy in x-dir.
 
         Args:
-            c (in): Courant number in flux form
+            c (in): product of x-dir wind on cell corners and timestep
```

Comment:
Yes, if I remember correctly this is commented in the Fortran code as being the "courant number in flux form". Nobody (including Lucas) could tell me what that meant, and it was one of my earliest questions about the code. I'm hoping this should be more clear, especially in adding that it's on cell corners which helps understand how this would be used to update the edge-center winds.